### DISCUSSION TITLE
Gnome-flashback not started

### DISCUSSION THREAD

**Discussion by tomfrancart** [regolith-linux/regolith-desktop#735] [Q&A]
I recently upgraded my Ubuntu version from 20.x to 22.x.
Now the lock screen function in regolith (super-esc) stopped working.
The issue seems to be that gnome-flashback is not started. Where in the regolith scripts is it supposed to be started and how can I fix it?

Some extra info:
* super-esc correctly executes `dbus-send --type=method_call --dest=org.gnome.ScreenSaver /org/gnome/ScreenSaver org.gnome.ScreenSaver.Lock`
* If I manually start gnome-flashback the screensaver is shown
* I have a custom i3 configfile in ~/.config/regolith/i3/config, but couldn't find any relevant settings relating to gnome-flashback
* ~/.config/regolith/Xresources doesn't seem to contain anything relevant
* I checked /usr/bin/regolith-session and regolith-session-init and they look fine, however they date from 2020

**Comment by kgilmer** [comment]
Hmm, I don't think Regolith will be functional if `gnome-flashback` is not running, but I've never tried to kill it.  I'm wondering if instead there is some other app or setting that's configured for that keybinding, and so i3 is not getting it?

To answer your question, here is where `gnome-flashback` is started: https://github.com/regolith-linux/regolith-session/blob/main/usr/bin/regolith-session#L16

**Comment by kgilmer** [comment]
@tomfrancart is it only `super-esc` that's giving you troubles?

**Comment by tomfrancart** [comment]
I found out that Regolith wasn't fully upgraded to version 2. After fixing that, the problem's solved. It would be nice if the Regolith upgrade would be automatic with the distribution upgrade. 
Thank you for your response!

**Comment by kgilmer** [comment]
Hi @tomfrancart , glad you got it fixed up.  Yes I believe that the Ubuntu upgrade process disables custom repositories by default, and they must be manually re-added after upgrade.  I guess the only way this problem would go away is if Regolith was an official Ubuntu DE..
